src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -0,0 +1,236 @@
+      div({ style: { margin: '0 0.5rem' } }, ['>']),
+      div({ style: Style.elements.sectionHeader }, [`Workflow ${workflowId}`])
+    ]),
+    _.isEmpty(workflow) ? centeredSpinner() : div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
+      makeSection('Workflow Status', [
+        div({ style: { lineHeight: '24px' } }, [makeStatusLine(style => statusIcon(status, style), status)])
+      ]),
+      makeSection('Workflow Timing', [
+        table({ style: { marginTop: '0.3rem', lineHeight: '20px' } }, [
+          tbody([
+            tr([td({ style: styles.sectionTableLabel }, ['Start:']), start ? td([Utils.makeCompleteDate(start)]) : 'N/A']),
+            tr([td({ style: styles.sectionTableLabel }, ['End:']), end ? td([Utils.makeCompleteDate(end)]) : 'N/A'])
+          ])
+        ])
+      ]),
+      makeSection('Links', [
+        div({ style: { display: 'flex' } }, [
+          h(Link, {
+            ...Utils.newTabLinkProps,
+            href: `${getConfig().jobManagerUrlRoot}/${workflowId}`,
+            style: { padding: '0.6rem' },
+            tooltip: 'Job Manager'
+          }, [
+            div({ style: { display: 'flex', alignItems: 'center' } }, [
+              icon('tasks', { size: 18 }),
+              span([' Job Manager'])
+            ])
+          ]),
+          h(Link, {
+            ...Utils.newTabLinkProps,
+            href: bucketBrowserUrl(`${bucketName}/${submissionId}/${workflowName}/${workflowId}`),
+            style: { padding: '0.6rem' },
+            tooltip: 'Execution directory'
+          }, [
+            div({ style: { display: 'flex', alignItems: 'center' } }, [
+              icon('folder-open', { size: 18 }),
+              span([' Execution Directory'])
+            ])
+          ]),
+          h(Link, { onClick: () => setShowLog(true), style: { padding: '.6rem' } }, [
+            icon('fileAlt', { size: 18, style: { marginRight: '0.5rem' } }),
+            ' View execution log'
+          ])
+        ])
+      ]),
+      makeSection('Call Statuses', [
+        workflow.calls && false ? statusCell(workflow) : div({ style: { padding: '0.6rem' } }, ['No calls have been started by this workflow.'])
+
+      ])
+    ]),
+    failures && h(Collapse, {
+      style: { marginBottom: '1rem' },
+      initialOpenState: true,
+      title: div({ style: Style.elements.sectionHeader }, [
+        'Workflow-Level Failures',
+        h(Link, {
+          style: { margin: '0 0.5rem' },
+          tooltip: 'Copy failures to clipboard',
+          onClick: withErrorReporting('Error copying to clipboard', async e => {
+            e.stopPropagation() // this stops the collapse when copying
+            await clipboard.writeText(JSON.stringify(failures, null, 2))
+            setFailureCopied(true)
+            await Utils.delay(1500)
+            setFailureCopied(undefined)
+          })
+        }, [icon(failureCopied ? 'check' : 'copy-to-clipboard')])
+      ])
+    }, [h(ReactJson, {
```

Comment:
TOL: Since we're rearranging the structure of the JSON and adding pseudo-messages, I wonder if it might be confusing to present this as a JSON structure. We don't have another easy way to display nested data right now, but something to think about going forward.

Reply:
I kind of agree. We played around with what element to use for this for a while. Ultimately we couldn't find a nice "nested tree structure" element which really matched what we needed and displaying as an almost-"straight-from-Cromwell" json viewer for now felt like a reasonable compromise. Worth noting that (IIRC) the "copy to clipboard" link still gives you the original-from-Cromwell text